modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,205 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-install_{context}"]
+= Installing an AWS Load Balancer (ALB) Operator
+Users that wish to install the AWS Load Balancer (ALB) Operator can do so as long as the following prerequisites are met:
+
+.Prerequisites
+
+* An existing {product-title} cluster with BYO-VPC installed in STS mode
+* Cluster-admin access
+* Access to modify the VPC and subnets of the created {product-title} cluster
+// * AWS account credentials
+* Configured {product-title} CLI
+* Configured Amazon Web Services (AWS) CLI
+* Configured OpenShift (oc) CLI
+* OpenShift Container Platform (OCP) 4.13 or later
+
+[NOTE]
+====
+When installing for use in an AWS Local Zone, the Local Zone is enabled for the account and the  Application Load Balancer (ALB) Operator is an available feature within the Local Zone. Additionally, the ability to use Amazon ELBv2 with {product-title} is subject to the availability of Application Load Balancers (ALB) and Network Load Balancers (NLB) in the AWS Availability Zones and Local Zones.
+====
+[IMPORTANT]
+====
+The Application Load Balancer (ALB) Operator supports creating Network Load Balancers (NLBs) on {product-title} clusters in either single-AZ or multi-AZ configurations. Application Load Balancers (ALBs) are supported in multi-AZ {product-title} clusters only, because Application Load Balancers (ALBs) require a multi-AZ configuration.
```

Comment:
ALB stands for different meaning so we should be careful when using it.
"Application Load Balancer" is concept of AWS and it is one of the three AWS Load Balancer types which are:

- Classic Load Balancer (CLB)
- Network Load Balancer (NLB)
- Application Load Balancer (ALB)

 And when we talking about the "ALB" Operator it should be AWS Load Balancer Operator, so the first statement should be 
```
The AWS Load Balancer Operator (ALBO) supports creating Network Load Balancers (NLBs) on ......
```
To avoid the confusion, I recommend to always use "AWS Load Balancer Operator (ALBO)" for the operator itself,  but not to use  "AWS Load Balancer (ALB) Operator".

Reply:
> ALB stands for different meaning so we should be careful when using it. "Application Load Balancer" is concept of AWS and it is one of the three AWS Load Balancer types which are:
> 
> * Classic Load Balancer (CLB)
> * Network Load Balancer (NLB)
> * Application Load Balancer (ALB)
> 
> And when we talking about the "ALB" Operator it should be AWS Load Balancer Operator, so the first statement should be
> 
> ```
> The AWS Load Balancer Operator (ALBO) supports creating Network Load Balancers (NLBs) on ......
> ```
> 
> To avoid the confusion, I recommend to always use "AWS Load Balancer Operator (ALBO)" for the operator itself, but not to use "AWS Load Balancer (ALB) Operator".

@lihongan, I understand what you are saying: However, I am using that terminology based on existing docs, as seen here: 
![image](https://github.com/openshift/openshift-docs/assets/122639474/2edb97f7-072b-4c7e-88c5-91d9c4c30e45)

Reply:
Yes, please don't abbreviate AWS Load Balancer to ALB. ALB refers to Application Load Balancer. 
Abbreviate AWS Load Balancer Operator to ALBO - that's Red Hat thing.

Reply:
Changed to ALBO based on recommendations.